distribution/lib/Standard/Database/0.2.32-SNAPSHOT/src/Data/Table.enso

```diff
@@ -171,7 +172,7 @@ type Table
              table.remove_columns (By_Name.new ["bar", "foo"])
 
        ## TODO [RW] default arguments do not work on atoms, once this is fixed,
-          the above should be replaced with just `Matching.Exact`.
+          the above should be replaced with just `By_Name`.
```

Comment:
these TODOs do not have links to tickets. Please add them. Every TODO has to have link to a relevant ticket.

Reply:
The link is below: `See: https://github.com/enso-org/enso/issues/1600`.
Or should we create a Pivotal ticket for this?

Reply:
The link is below: `See: https://github.com/enso-org/enso/issues/1600`.
Or should we create a Pivotal ticket for this?

Reply:
The link is below: `See: https://github.com/enso-org/enso/issues/1600`.
Or should we link to a Pivotal ticket instead? (I think one does not exist for this issue currently, so shall it be created?)